encrypted-media-respec.html

```diff
@@ -318,7 +318,7 @@
           </p>
           <p>Each Key Session is associated with a single <a>MediaKeys</a> object, and only media element(s) associated with that object may access key(s) associated with the session.
           Other <a>MediaKeys</a> objects, <a def-id="cdm"></a> instances, and media elements MUST NOT access the key session or use its key(s).
-          Key sessions and the keys they contain are no longer <a def-id="usable-for-decryption"></a> when the <a def-id="session-close-algorithm">session is closed</a>, including when the <a>MediaKeySession</a> object is destroyed.
+          Key sessions and the keys they contain are no longer <a def-id="usable-for-decryption"></a> when the <a def-id="session-closed-algorithm">session is closed</a>, including when the <a>MediaKeySession</a> object is destroyed.
```

Comment:
Yes, you agree with my suggestion? I don't follow how the second sentence applies here.

Looking at this again, the Session Closed algorithm is _not_ run `including when the <a>MediaKeySession</a> object is destroyed` because the object no longer exists. Either we should remove the session-closed-algorithm link or convert "including" to "or" or similar.